subiquity/server/controllers/filesystem.py

```diff
@@ -1800,6 +1811,63 @@ async def v2_core_boot_encryption_features_GET(
 
         raise StorageInvalidUsageError("no suitable variation for core boot")
 
+    async def v2_core_boot_fix_encryption_support_POST(
+        self, data: CoreBootFixEncryptionSupport
+    ) -> None:
+        # Actions that are handled by Subiquity, not snapd.
+        local_actions = {
```

Comment:
nitpick: I'm not sure local is the best term.  What do you think about `subiquity_actions`?